frontend/packages/kubevirt-plugin/src/selectors/vm/selectors.ts

```diff
@@ -96,3 +102,27 @@ export const getCloudInitUserData = (vm: VMKind) =>
 
 export const hasAutoAttachPodInterface = (vm: VMKind, defaultValue = false) =>
   _.get(vm, 'spec.template.spec.domain.devices.autoattachPodInterface', defaultValue);
+
+export const getCDRoms = (vm: VMLikeEntityKind) => getDisks(vm).filter((device) => !!device.cdrom);
+
+export const getContainerImageByDisk = (vm: VMLikeEntityKind, name: string) =>
+  getVolumeContainerImage(getVolumes(vm).find((vol) => name === vol.name));
+
+export const getPVCSourceByDisk = (vm: VMLikeEntityKind, diskName: string) =>
+  getVolumePersistentVolumeClaimName(getVolumes(vm).find((vol) => vol.name === diskName));
+
+export const getURLSourceByDisk = (vm: VMLikeEntityKind, name: string) =>
+  _.get(
```

Comment:
All newly added selectors should use direct access (discussed in other PRs).
Example:
```
return myDVTemplate && myDVTemplate.spec && myDVTemplate.spec.source && myDVTemplate.spec.source.http;
```